tests/datadriven/test_variants.py

```diff
@@ -56,11 +57,77 @@ def getDataModelClass(self):
     def getProtocolClass(self):
         return protocol.GAVariantSet
 
-    def verifyVariantsEqual(self, gaVariants, pyvcfVariants):
+    def _compareTwoFloats(self, a, b):
+        afloat = float(a)
+        bfloat = float(b)
+        return (abs(afloat - bfloat) <= 0.001 * abs(bfloat))
```

Comment:
Not loving the `0.001` magic number. Also, "compare" in the method signature suggests a comparison style return `(<,==,>)` as opposed to a bool. Suggest something like:
```
def floatsAgreeWithinTolerance(self, a, b, decimalPoints = 9):
    [...]
    return (abs(afloat - bfloat) <= 10**(-decimalPoints) * abs(bfloat))
```
If we're just worried about floating point math-induced differences, the tolerance can be much tighter than 3 sig digs.
Here's something I found that uses `FLT_EPSILON` to address this issue:
https://randomascii.wordpress.com/2012/02/25/comparing-floating-point-numbers-2012-edition/

Apologies for the long and unwieldy suggested method name: I'm coming off an Objective-C programming project(I had to stymie my impulse to call it `[Utils compareFloat: a withFloat: b toSignificantDigits: decimalPoints];`). I welcome other suggestions for naming such a method.